src/Library/Platform/Interface/PlatformOpenGLContext.h

```diff
@@ -7,6 +7,7 @@ class PlatformOpenGLContext {
     virtual bool bind() = 0;
     virtual bool unbind() = 0;
 
+    virtual void startOverlayFrame() = 0;
```

Comment:
I don't know well the abstraction system put in place between SDL/OpenGL so let me know if this new method is completely out of place. Let me know if you have a better alternative.

Reply:
Looks commando-style to me X).

Let's not introduce a Platform -> ImGui dependency.

I'm OK with `ImGuiRenderer` (or however you'll call it) relying on `Platform` being implemented on top of SDL.